index.js

```diff
@@ -35,3 +35,8 @@ HTMLReactParser.attributesToProps = attributesToProps;
 // support CommonJS and ES Modules
 module.exports = HTMLReactParser;
 module.exports.default = HTMLReactParser;
+
+// better support for ESM build tools
+exports.domToReact = domToReact;
```

Comment:
Thanks for opening this PR @davidbailey00! Unfortunately these changes will cause the UMD build to break.

You can verify this by running `npm run build`:

```sh
$ npm run build

> html-react-parser@1.1.2 build /Users/mark/Code/npm/html-react-parser
> rollup --config


index.js → dist/html-react-parser.js...
(!) Mixing named and default exports
https://rollupjs.org/guide/en/#outputexports
The following entry modules are using named and default exports together:
index.js

Consumers of your bundle will have to use chunk['default'] to access their default export, which may not be what you want. Use `output.exports: 'named'` to disable this warning
created dist/html-react-parser.js in 515ms

index.js → dist/html-react-parser.min.js...
(!) Mixing named and default exports
https://rollupjs.org/guide/en/#outputexports
The following entry modules are using named and default exports together:
index.js

Consumers of your bundle will have to use chunk['default'] to access their default export, which may not be what you want. Use `output.exports: 'named'` to disable this warning
created dist/html-react-parser.min.js in 1.1s
```

And then opening `examples/index.html`:

```sh
$ open examples/index.html
```

You should see the error in your browser `Console`:

```
Uncaught TypeError: window.HTMLReactParser is not a function
    at renderOutput (index.html:12)
    at index.html:19
```

Reply:
Thanks for pointing out this issue as I didn't spot it myself! I've pushed an alternative approach in the form of adding a new entry point for ES modules, which I've verified also works as expected with Snowpack and doesn't change the CJS approach.

> Any reason you can't do this?

I can do this yes, or the ESM equivalent:

```js
import HTMLReactParser from 'html-react-parser';
const { domToReact, htmlToDOM } = HTMLReactParser;
```

However this is a little annoying when I don't actually want to use the default export. In addition, it doesn't line up with the type definitions which expect `domToReact` etc to be imported as a named import, and don't provide type definitions for those as properties of the default import.